### DISCUSSION TITLE
Buffers are not recovered after restarting emacs

### DISCUSSION THREAD

**Discussion by linwaytin** [alphapapa/activities.el#124] [General]
I'm using this package with emacs daemon mode. When I restart the emacs session, only the buffer which was focused in the previous session is reopened. 

This package is the only session manager I am using. Is this the expected behavior?

**Comment by alphapapa** [comment]
Hello again,

It seems like you may be expecting this library to do something it isn't intended to do (at least, not in its current iteration).  Please see the documentation I wrote, which explains carefully what it does.  If you read it and still have questions, let me know.  Thanks.
